ariadne/src/Ariadne/Config/Cardano.hs

```diff
@@ -1,29 +1,333 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
1. Orphan `Eq` instances should be moved to `Ariadne.Cardano.Orphans`.
2. Orphan `Default` instances are not needed. `def :: X` can be turned into `defX :: X`. This type class doesn't give much benefit in this case. Moreover, it may cause some trouble if someone else defines `instance Default X` somewhere.